autode/calculations/executors.py

```diff
@@ -445,7 +446,7 @@ def _step_size(self) -> float:
 
     @property
     def _opt_trajectory_name(self) -> str:
-        return f"{self.name}_opt_trj.xyz"
+        return f"{self.name}_opt_trj.zip"
```

Comment:
sorry stupid question: there's no xyz trajectory for an optimisation anymore? how annoying is this for debugging? will mean this is a breaking change and needs to be targeted at v1.5.0(?)

Reply:
@t-young31 the xyz trajectory can now be written with `optimiser.print_geometries()`, I don't think its a big change since even before the `optimiser.save()` needed to be called to write the trajectory. I edited that part so that the trajectory name does not have extension, and the xyz trajectory is printed after the run.